src/components/Login/Login.jsx

```diff
@@ -110,12 +110,30 @@ export class Login extends React.Component {
                         <span className={styles.letter} data-letter="t">t</span>
 
                     </div>
-                    <form className={styles.loginForm} onSubmit={this.sendLogin.bind(this)}>
-                        <h3 className={styles.errorText}>{this.state.error}</h3>
-                        <h2 className={styles.formHeader}>Type a username to enter the game:</h2>
-                        <input className={styles.loginInput} style={this.getInputStyle()} value={this.state.value} placeholder="Enter username" onChange={this.handleChange.bind(this)} autoFocus/>
-                        <button className={styles.loginButton} type="submit">Log in</button>
-                    </form>
+                    <div className={styles.ufo}>
```

Comment:
Might be worth you moving the monster into it's own component so that you learn how to make a component, I can show you if you want

Reply:
cool, tomorrow maybe?